docs/src/languages/nodejs/_index.md

```diff
@@ -128,33 +128,95 @@ npm prune --userconfig .npmrc && npm install --userconfig .npmrc
 
 This means you can specify configuration in a `.npmrc` file in [your app root](../../create-apps/app-reference.md#root-directory).
 
-### Use yarn as a package manager
+### Use Yarn as a package manager
 
-To switch to yarn to manage dependencies, follow these steps:
+To switch to Yarn to manage dependencies, follow these steps:
 
-1. Switch to a build flavor of `none` (so packages aren't installed with npm):
+1. Turn off the default use of npm:
 
    ```yaml {location=".platform.app.yaml"}
    build:
        flavor: none
    ```
 
-2. Add yarn as a global dependency:
+2. Specify the version of Yarn you want:
+
+   ```json {location="package.json"}
+   {
+     ...
+     "packageManager": "yarn@3.2.1"
+   }
+   ```
+
+What you do next depends on the version of Yarn and Node.js you want.
+
+{{< codetabs >}}
+
+---
+title=Yarn 3.x and Node.js 16
```

Comment:
:question:
Should the [minor version be added](https://github.com/nodejs/corepack#default-installs)? Or maybe a link to corepack's setup information?

```suggestion
title=Yarn 3.x and Node.js 16.9
```

Reply:
Users can't specify minor versions. Only major ones. So they can't use version 16.9 even if they want to.

Not sure what setup information you'd want to have. It's automatically enabled for 16.10+.